environments/docker.toml

```diff
@@ -336,11 +336,13 @@
         repository = "github.com/makerdao/vdb-mcd-transformers"
```

Comment:
Do we also need to add these new contract versions to the storage transformers list as well?

Reply:
Also, I think we'd need new storage transformer initializers for those as well.

Reply:
will be included when #302 and #303 are merged 👍